doc/source/ray-air/examples/analyze_tuning_results.ipynb

```diff
@@ -0,0 +1,918 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c62191af",
+   "metadata": {},
+   "source": [
+    "# Analyzing results from hyperparameter tuning\n",
+    "In this example we'll run a hyperparameter tuning experiment and analyze the results.\n",
+    "\n",
+    "What we'll cover:\n",
+    "- How to load data from an Sklearn example dataset\n",
+    "- How to initialize an XGBoost trainer\n",
+    "- How to define a search space for regular XGBoost parameters and for data preprocessors\n",
+    "- How to fetch the best obtained result from the tuning run\n",
+    "- How to fetch a dataframe to do further analysis on the results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "41abda7b",
+   "metadata": {},
+   "source": [
+    "First we start with some imports. We'll use the [Covertype dataset](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_covtype.html#sklearn-datasets-fetch-covtype) provided from sklearn to train a multiclass classification task using XGBoost."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "d45b4f69",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from sklearn.datasets import fetch_covtype\n",
+    "\n",
+    "import ray\n",
+    "from ray import tune\n",
+    "from ray.ml import RunConfig\n",
+    "from ray.ml.preprocessors import StandardScaler\n",
+    "from ray.ml.train.integrations.xgboost import XGBoostTrainer\n",
+    "from ray.tune.tune_config import TuneConfig\n",
+    "from ray.tune.tuner import Tuner"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a93b242c",
+   "metadata": {},
+   "source": [
+    "We'll define a utility function to create a Ray Dataset from the Sklearn dataset. We expect the target column to be in the dataframe, so we'll add it to the dataframe manually.\n",
+    "We don't do any test/validation splitting here - pelase check out our other examples for this."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "3875df98",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-05-09 17:25:30,768\tINFO services.py:1484 -- View the Ray dashboard at \u001B[1m\u001B[32mhttp://127.0.0.1:8265\u001B[39m\u001B[22m\n"
+     ]
+    }
+   ],
+   "source": [
+    "def get_training_data() -> ray.data.Dataset:\n",
```

Comment:
This won't work with `tuner.resume` though. Do we want the example to showcase that?

Reply:
It should with `lambda:` right? I think it's fine to leave out, this example focuses on analysis, not on the problem